styles/main.css

```diff
@@ -0,0 +1,419 @@
+
+:root {
+  --gray-one: #cfcfcf;
+  --big-border: 1.5rem solid #ececec;
+}
+
+.background-color {
+  background-color: #f72210;
+}
+
+.header-logo {
+  margin-top: -0.5rem;
+  width: 8rem;
+}
+
+.navbar {
+  padding: 0 0.1rem;
+  margin-bottom: -0.7rem;
+}
+
+.btn {
+  padding: 0.2rem 0.5rem;
+  font-size: 0.7rem;
+  background-color: #ffe400;
+  border-radius: 3rem;
+  margin: auto 0.6rem;
+}
+
+i {
+  color: rgba(255, 255, 255, 0.699);
+}
+
+.navbar p {
+  margin-bottom: 0.1rem;
+}
+
+.container-fluid {
+  padding: 0;
+}
+
+.ul-inline-block li {
+  padding-left: 1.3%;
+  display: inline;
+}
+
+.ul-inline-block li + li {
+  border-left: #9b9b9b solid 1px;
+}
+
+.ul-inline-block {
+  width: 85%;
```

Comment:
Please try to use bootstrap for this project. If a goal can be achieved from bootstrap, don't use CSS.